src/main/resources/com/puppycrawl/tools/checkstyle/checks/annotation/messages_de.properties

```diff
@@ -2,7 +2,7 @@ annotation.incorrect.style=Die Annotation muss dem Stil ''{0}'' folgen.
 annotation.location=Die Annotation ''{0}'' hat eine unerwartete Einrückungstiefe von {1} (erwartet: {2}).
 annotation.location.alone=Die Annotation ''{0}'' sollte in einer eigenen Zeile stehen.
 annotation.missing.deprecated=Eine Deprecated-Annotation (@java.lang.Deprecated) und ein Deprecated-Javadoc-Tag (@deprecated) mit Beschreibung müssen immer zusammen verwendet werden.
-annotation.missing.override=Wenn das '{'@inheritDoc'}'-Javadoc-Tag verwendet wird, muss eine Override-Annotation (@java.lang.Override) vorhanden sein.
+annotation.missing.override=Wenn das '@inheritDoc'-Javadoc-Tag verwendet wird, muss eine Override-Annotation (@java.lang.Override) vorhanden sein.
```

Comment:
here and in other translations below quote is not escaped

Reply:
Fixed it.